diff --git a/node_modules/react-native-confirmation-code-field/esm/CodeField.js b/node_modules/react-native-confirmation-code-field/esm/CodeField.js
index 5a0e131..92fe895 100644
--- a/node_modules/react-native-confirmation-code-field/esm/CodeField.js
+++ b/node_modules/react-native-confirmation-code-field/esm/CodeField.js
@@ -1,4 +1,5 @@
-import { Platform, TextInput, View, } from 'react-native';
+import { Platform, View } from 'react-native';
+import { BottomSheetTextInput } from '@gorhom/bottom-sheet';
 import React, { forwardRef, } from 'react';
 import { getStyle, getSymbols } from './utils';
 import { useFocusState } from './useFocusState';
@@ -8,7 +9,7 @@ const autoComplete = Platform.select({
     android: 'sms-otp',
     default: 'one-time-code',
 });
-function CodeFieldComponent({ rootStyle, textInputStyle, onBlur, onFocus, value, renderCell, cellCount = DEFAULT_CELL_COUNT, RootProps = {}, RootComponent = View, InputComponent = TextInput, ...rest }, ref) {
+function CodeFieldComponent({ rootStyle, textInputStyle, onBlur, onFocus, value, renderCell, cellCount = DEFAULT_CELL_COUNT, RootProps = {}, RootComponent = View, InputComponent = BottomSheetTextInput, ...rest }, ref) {
     const focusState = useFocusState(onBlur, onFocus);
     const cells = getSymbols(value || '', cellCount).map((symbol, index, symbols) => {
         const isFirstEmptySymbol = symbols.indexOf('') === index;
